draft-ietf-mls-protocol.md

```diff
@@ -1926,15 +1922,16 @@ the next epoch change anyway. ]]
 A member of the group creates a Commit message and the corresponding Welcome
 message at the same time, by taking the following steps:
 
-* Construct an initial Commit object with `updates`, `removes`, `adds`, and
-  `ignored` fields populated from Proposals received during the current epoch,
-  and an empty `path` field.
+* Construct an initial Commit object with `updates`, `removes`, and `adds`
+  fields populated from Proposals received during the current epoch, an Update
+  proposal for the member's own leaf in `committer_update`, and an empty `path`
+  field.
 
 * Generate a provisional GroupContext object by applying the proposals
   referenced in the initial Commit object in the order provided, as described in
   {{proposals}}. Add proposals are applied left to right: Each Add proposal is
   applied at the leftmost unoccupied leaf, or appended to the right edge of the
-  tree if all leaves are occupied.
+  tree if all leaves are occupied. The committer's own Update is applied last.
```

Comment:
It would be good to make clear that the Update in question is the `committer_update`.  Also, could you please check to make sure that DirectPath no longer touches the leaf?  Since we have split out Update and `committer_update`, DirectPath should start at the parent of the Committer.

Reply:
https://github.com/mlswg/mls-protocol/pull/294